packages/css/src/card/card.css

```
@@ -85,7 +88,7 @@
   .hds-card__body-action,
   .hds-card__body-action-arrow {
     display: inline-block;
-    margin-top: var(--hds-spacing-16-20);
+    margin-top: 16px;
```

We use literals if we can avoid it.

```suggestion
    margin-top: var(--hds-spacing-16);
```